diff -aurN jackEQ-0.3.6.orig/src/plugin.c jackEQ-0.3.6/src/plugin.c
--- jackEQ-0.3.6.orig/src/plugin.c	2003-08-13 08:27:11.000000000 +0200
+++ jackEQ-0.3.6/src/plugin.c	2003-12-02 22:28:27.000000000 +0100
@@ -13,7 +13,7 @@
 	if (getenv("LADSPA_PATH")) {
 		plugin_path = strdup(getenv("LADSPA_PATH"));
 	} else {
-		plugin_path = strdup("/usr/local/lib/ladspa:/usr/lib/ladspa");
+		plugin_path = strdup("/usr/lib/ladspa");
 	}
 }
 
